crates/subspace-farmer/src/bin/subspace-farmer/commands/benchmark.rs

```diff
@@ -188,7 +188,7 @@ where
                             reward_address: single_disk_farm_info.public_key(),
                             slot_info: SlotInfo {
                                 slot_number: 0,
-                                global_challenge,
+                                global_challenge: Blake3Hash::from_array(global_challenge),
```

Comment:
```suggestion
                                global_challenge: Blake3Hash::from(global_challenge),
```
or (something that you used in other places, but something that I generally try to avoid):
```suggestion
                                global_challenge: global_challenge.into(),
```

Reply:
I tried this, but it causes type inference issues, so I think we'd have to also change the closure to take `|global_challenge: [u8; 32]|`.

Reply:
I think you'll need `rand::random::<[u8; 32]>` above or you can create the `GlobalChallenge` there directly.